Sources/Input.swift

```diff
@@ -23,7 +23,7 @@ public class Input {
         - Parameter message: message to be printed before accepting input (e.g. "Name: ")
         - Throws: Input.PipeUserInputOverlapError if the user piped data and this method was called
     */
-    public class func awaitInput(message message: String?) throws -> String {
+    public class func awaitInput(_ message: String?) throws -> String {
```

Comment:
`awaitInput(message: String?)` and same for the rest of the methods in this file